index.html

```diff
@@ -22,7 +21,7 @@
     <!-- build:js assets/bundle.js -->
     <!-- bower:js -->
     <script src="/bower_components/jquery/dist/jquery.js"></script>
-    <script src="/bower_components/bootswatch-dist/js/bootstrap.js"></script>
+    <script src="/bower_components/bootstrap-sass/assets/javascripts/bootstrap.js"></script>
```

Comment:
I didn't modified it manually, but wiredep-ed since I removed bootswatch-dist.
and I think it is a good idea to include the result of wiredep in the commit.
I just didn't know that commit is not needed.